packages/core/src/domain/console/consoleObservable.ts

```diff
@@ -6,9 +6,17 @@ import { ConsoleApiName } from '../../tools/display'
 import { callMonitored } from '../../tools/monitor'
 
 export interface ConsoleLog {
+  /* Error message created from the concatenation of console parameters */
   message: string
+  /* The console API generating this log (debug, info, warn, error, ...) */
   api: ConsoleApiName
+  /* The specific type of the first Error object present in console parameters (SyntaxError, RangeError, ...) */
+  kind?: string
+  /* The stack trace provided by the first Error object present in console parameters */
   stack?: string
+  /* The message included in the first Error object present in console parameters */
+  errorMessage?: string
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏I think this might be a breaking change (using error.message instead of the full console message). Maybe we could remove this change from this PR?

Reply:
FMU, it still uses the full message in log.message but now use the error message in log.error.message.
However, I'd be in favour of focusing the scope of this PR to only the logger APIs and address other improvements in a following PR to also discuss:

- usage of error.message on other error sources as well
- stacktrace computation of error instance for other console APIs

Reply:
Removed the commit for consoleObservable - to be handled in another PR.